emmet-core/emmet/core/vasp/validation.py

```diff
@@ -5,6 +5,7 @@
 from pydantic import BaseModel, Field, PyObject
 from pymatgen.core import Structure
 
+from emmet.builders import EmmetBuildSettings
```

Comment:
```suggestion
```

No, don't import this here. There is a settings type for core emmet in `emmet.core.settings` if needed.